README.md

```diff
@@ -26,11 +26,17 @@ LOCATIONS_INSIDE_PRISON_API_CLIENT_SECRET=
 Start the database and other required services via docker-compose with:
 
 ```shell
-docker compose -f docker-compose-local.yml up
+docker compose up
 ```
 
 Then run the API.
 
+### Running the whole setup in docker
+
+```shell
+docker compose --profile include-api up
```

Comment:
You can also do
```
docker compose --profile include-api up --build
```
to make sure it does a fresh build when running it, or even
```
docker compose --profile include-api up --build --push
```
to keep the deps up-to-date

Reply:
I'll add to next PR!